exchanges/kucoin/kucoin_types.go

```diff
@@ -2261,6 +2292,27 @@ type MarginPairConfigs struct {
 	Items     []MarginPairConfig   `json:"items"`
 }
 
+// MarginActiveSymbolDetail represents an active high frequency margin symbols information
+type MarginActiveSymbolDetail struct {
+	SymbolSize int64    `json:"symbolSize"`
+	Symbols    []string `json:"symbols"`
+}
+
+// MarginInterestRecords represents a cross/isolated margin interest records
+type MarginInterestRecords struct {
+	Timestamp   int64 `json:"timestamp"`
+	CurrentPage int64 `json:"currentPage"`
+	PageSize    int64 `json:"pageSize"`
+	TotalNum    int64 `json:"totalNum"`
+	TotalPage   int64 `json:"totalPage"`
+	Items       []struct {
```

Comment:
Suggestion: Would prefer at a wrapper level we shy away from using anonymous structs as they are hard to deal with from a library usage perspective when you want to test.

Reply:
All occurrences of the specified type have been changed as requested 🫡